### ISSUE TITLE
Detections bounding box whole image

### ISSUE THREAD

**Issue by dopplesoldner** [weiliu89/caffe#204]
Hi

I've managed to train an SSD using my custom dataset and achieved test MAP of 75%.

However, when i test is on same images using ssd_detect.ipynb, i observe bounding boxes that cover the entire image.. i.e minx=miny=0,  maxx = maxy = 1.

Any ideas where I might be going wrong?

Sample output 

`
detections = net.forward()['detection_out']
array([[[[ 0.        ,  2.        ,  0.24353264,  0.52570081,  0.25105521,
           0.61999714,  0.34230915],
         [ 0.        ,  2.        ,  0.02272068,  0.08548105,  0.18355547,
           0.12004389,  0.21929528],
         [ 0.        ,  5.        ,  1.        ,  0.        ,  0.        ,
           0.        ,  1.        ],
         [ 0.        ,  5.        ,  1.        ,  1.        ,  0.        ,
           1.        ,  1.        ],
         [ 0.        ,  5.        ,  1.        ,  0.        ,  1.        ,
           1.        ,  1.        ],
         [ 0.        ,  5.        ,  1.        ,  1.        ,  1.        ,
           1.        ,  1.        ],
         [ 0.        ,  5.        ,  0.01205588,  0.72067869,  0.2859889 ,
           0.8308301 ,  0.38130215]]]], dtype=float32)`

**Comment by weiliu89** [comment]
Not sure why. Was it only on class 5?

**Comment by dopplesoldner** [comment]
No, every image is a new class, I've tried the debug ipython notebook as well and my data seems to be loaded correctly. The training process also shows MAP > 0.7.
The only thing I can think of is that my objects are quite small compared to the size of the frame.
